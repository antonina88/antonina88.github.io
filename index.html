<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Code examples by Antonina Kukhta</title>
</head>
<body>
	<header class="page-header">
	    <h1 class="project-name">Code examples</h1>
	    <h2 class="project-tagline"></h2>
	</header>

	<section class="main-content">
	    <h3>Pages</h3>
	    <p><a href="https://antonina88.github.io/golden/index.html">Golden</a></p>
	    <p><a href="https://antonina88.github.io/convert/index.html">Convert</a></p>
	    <p><a href="https://antonina88.github.io/music/index.html">Music</a></p>
	    <p><a href="https://antonina88.github.io/forecast/index.html">Forecast</a></p>
	</section>
</body>
</html>