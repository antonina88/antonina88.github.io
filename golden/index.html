<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1"/>
	<title>Golden page</title>
	<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700" rel="stylesheet">
	<link rel="stylesheet" href="./css/styles.css">
</head>
<body>
	<div class="wrapper">
		<header class="header">
			<div class="header__container">
				<a href="index.html">
					<img class="logo" src="./img/logo.png" alt="img logo" />
				</a>
				<button class="btn-menu"></button>
				<nav class="nav-menu">
					<button class="btn-close"></button>
					<ul class="main-menu">
						<li class="main-menu__item">
							<a class="main-menu__link" href="#">Home</a>
						</li>
						<li class="main-menu__item">
							<a class="main-menu__link" href="#services">Services</a>
						</li>
						<li class="main-menu__item">
							<a class="main-menu__link" href="#portfolio">Portfolio</a>
						</li>
						<li class="main-menu__item">
							<a class="main-menu__link" href="#about">About</a>
						</li>
						<li class="main-menu__item">
							<a class="main-menu__link" href="#contacts">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="header__info">
				<p class="header__text header__text--italic">welcome to our studio!</p>
				<p class="header__text header__text--bold">it's nice to meet you</p>
				<button class="header__btn">tell me more</button>
			</div>
		</header>
		<section class="services" id="services">
			<h2 class="caption">services</h2>
			<p class="services__description">Proin iaculis purus consequat sem cure.</p>

			<div class="services__container">
				<div class="services__item">
					<div class="services__icon services__icon--ecommerce"></div>
					<h3 class="services__caption">E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services__item">
					<div class="services__icon services__icon--web"></div>
					<h3 class="services__caption">Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services__item">
					<div class="services__icon services__icon--security"></div>
					<h3 class="services__caption">Web Security</h3>
					<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</section>
		<section class="portfolio" id="portfolio">
			<h2 class="caption">Our portfolio</h2>
			<p class="description">Proin iaculis purus consequat sem cure.</p>
			<div class="projects">
				<div class="projects__item">
					<div class="projects__image"></div>
					<h3 class="projects__caption">Ebony & Ivory</h3>
					<p class="projects__description">Branding</p>
				</div>
				<div class="projects__item">
					<div class="projects__image"></div>
					<h3 class="portfolio__caption">Smart Stationary</h3>
					<p class="projects__caption">Print Design</p>
				</div>
				<div class="projects__item">
					<div class="projects__image"></div>
					<h3 class="projects__caption">Clever Poster</h3>
					<p class="projects__description">Print Design</p>
				</div>
				<div class="projects__item">
					<div class="projects__image"></div>
					<h3 class="projects__caption">Vinyl Record</h3>
					<p class="projects__description">Product Mock-Up</p>
				</div>
				<div class="projects__item">
					<div class="projects__image"></div>
					<h3 class="projects__caption">Treehouse Template</h3>
					<p class="projects__description">Web Design</p>
				</div>
				<div class="projects__item">
					<div class="projects__image"></div>
					<h3 class="projects__caption">Burned Logo</h3>
					<p class="projects__description">Branding</p>
				</div>
			</div>
		</section>
		<section class="about" id="about">
			<h2 class="caption">About us</h2>
			<p class="description">Proin iaculis purus consequat sem cure. </p>
			<div class="stories">
				<div class="stories__item">
					<div>
						<p class="stories__date">JULY 2010</p>
						<h3 class="stories__caption">Our Humble Beginnings</h3>
						<p class="stories__description">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
					</div>
					<div class="stories__image"></div>
				</div>
				<div class="stories__item">
					<div class="stories__image"></div>
					<div>
						<p class="stories__date">January 2011</p>
						<h3 class="stories__caption">Facing Startup Battles</h3>
						<p class="stories__description">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
				<div class="stories__item">
					<div>
						<p class="stories__date">December 2012</p>
						<h3 class="stories__caption">Enter The Dark Days</h3>
						<p class="stories__description">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
					</div>
					<div class="stories__image"></div>
				</div>
				<div class="stories__item">
					<div class="stories__image"></div>
					<div>
						<p class="stories__date">february 2014</p>
						<h3 class="stories__caption">Our Triumph</h3>
						<p class="stories__description">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
				<div class="stories__continue">our story continue</div>
			</div>
		</section>
		<div class="contacts" id="contacts">
			<h2 class="caption caption--light">contact us</h2>
			<p class="description">Proin iaculis purus consequat sem cure. </p>

			<form class="form">
				<div class="form__container">
					<input class="form__input" type="text" placeholder="your name" />
					<input class="form__input" type="text" placeholder="your e-mail" />
					<input class="form__input" type="text" placeholder="subject" />
				</div>
				<textarea class="form__message" placeholder="your message"></textarea>
				<input type="submit" class="form__btn" />
			</form> 
		</div>

		<footer class="footer">&copy; Copyright 2014 FreebiesXpress.com</footer>
	</div>

	<script src="./js/jquery-3.3.1.min.js"></script>
	<script async src="./js/main.js"></script>
</body>
</html>